release.sh

```
@@ -0,0 +1,232 @@
+# git push origin "$TAG_NAME"
+
+# print_info "🎉 Release completed successfully!"
+# print_info "Version : $NEW_VERSION"
+# print_info "Tag     : $TAG_NAME"
+
+
+
+
+
+
+#!/bin/bash
```

Shellcheck (0.11.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)